lib/crystalball/source_diff/formatting_checker.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Crystalball
+  class SourceDiff
+    # Determinates if file_diff's patch contains changes for whitespaces or comments only
+    module FormattingChecker
+      class << self
+        # Returns `true` if file_diff's patch contains changes for whitespaces or comments only
+        #
+        # @param [Crystalball::SourceDiff::FileDiff] file_diff
+        # @return [TrueClass, FalseClass]
```

Comment:
I think yard supports `@return [Boolean]`, no need to specify `TrueClass, FalseClass`